### PR TITLE
Branch/no tick interp

### PR DESCRIPTION
**Pull request by scottnc27603**
This is based off of Rob’s view interpolation.  I tested by calling translateBy from the console for the duck and using the test/JSONLoadingTest applications.  It’s a little harder to be sure the keyframe animations are interpolating correctly.

### CODE REVIEW

File: support/proxy/vwf.example.com/node3.vwf.yaml [hunk 1]
```diff
@@ -274,45 +274,6 @@ properties:
...
-
-  speed: 1
-
-  ## Playing value
-  ## 
-  ## @name node3.vwf#playing
-  ## @property
-
-  playing: false
-
```
Comment (allisoncorey) [inline support/proxy/vwf.example.com/node3.vwf.yaml]: Do we no longer need these properties? (speed, looping, playing)

File: support/client/lib/vwf/view/threejs.js [hunk 2]
```diff
@@ -172,9 +172,9 @@ define( [ "module",
                 setActiveCamera.call(this, this.state.scenes[ this.kernel.application() ].camera.ID);
             }
         
-            if(this.state.nodes[childID] && this.state.nodes[childID].threeObject instanceof THREE.Object3D) {
-                this.nodes[childID] = {id:childID,extends:childExtendsID};
-            }
+            // if ( this.state.nodes[childID] && this.state.nodes[childID].threeObject instanceof THREE.Object3D ) {
+            //     this.nodes[childID] = {id:childID,extends:childExtendsID};
+            // }
         },
```
Comment (allisoncorey) [inline support/client/lib/vwf/view/threejs.js]: If this is no longer needed, I vote to go ahead and remove it rather than comment it out.

**Comment by allisoncorey** [comment]
We should probably go ahead and merge this into the persistence-prerequisites branch as well to confirm it solves the problems we're seeing there with view interpolation.

File: support/proxy/vwf.example.com/node3.vwf.yaml (hunk 1 above)
Comment (davideaster) [inline support/proxy/vwf.example.com/node3.vwf.yaml]: I searched for them yesterday and didn't find any references (except in `control-old.vwf`, which doesn't seem to be used either). They've been replaced by `animationPlaying`, etc. in `animation.vwf`.

**Comment by davideaster** [comment]
Agreed. It should be on top of `branch/tickless-model` since that's where the breaking change occurred.

**Comment by davideaster** [comment]
By the way, here's how I've been recreating `branch/persistence-prerequisites` after changes to the underlying branches:

    git checkout -B branch/persistence-prerequisites 06408c0  # 06408c0 is the common base of the branches
    git merge branch/tickless-model branch/absolute-application-url branch/component-relative-resources branch/modern-ids branch/reload-on-disconnect
    git push -f origin

**Comment by allisoncorey** [comment]
This branch has been merged into branch/tickless-model, in PR #450 
Closing.

**Pull request closed without merging**